tox.ini

```diff
@@ -12,8 +12,7 @@ deps =
     -rtest_requirements.txt
 
 commands =
-    django-admin.py test auth_backends --settings=test_settings --with-coverage --cover-package=auth_backends
-    coverage report
+    py.test {posargs}
```

Comment:
Just call `pytest`. `py.test` is the old way; the [docs](http://doc.pytest.org/en/latest/) recommend `pytest`.